Add blog post for restic 0.17.1 (#40)
restic/restic.net: content/blog/restic-0.17.1-released.md

---
title: "Restic 0.17.1 Released"
date: 2024-09-03T00:00:00+02:00
---

We are happy to announce the release of [restic 0.17.1](https://github.com/restic/restic/releases/v0.17.1)!

> This release contains various bugfixes and small enhancements. The most important are the following:

Reviewer: ```suggestion
This release contains various bug fixes and small enhancements. The most important ones are the following:
```

Author: changed. Thanks!

> - The backup command now always stores files and directories even if some metadata could not be retrieved.

Reviewer: ```suggestion
- The backup command now also stores files and directories for which some metadata could not be retrieved.
```

Author: also changed.
